src/components/LectureEventModal/LectureEventModal.tsx

```diff
@@ -83,6 +85,18 @@ const LectureEventModal = ({
             </div>
           ))}
         </div>
+        {lectureEvent.title.toLowerCase().includes('anglų kalba') && (
+          <div className={styles.lectureInfoContainer}>
+            {' '}
```

Comment:
remove this empty space, it shouldn't affect anything, if I'm wrong and it affects something, keep it